src/Apps/Legion/Models/Wizard.types.ts

```diff
@@ -45,7 +45,7 @@ export interface IViewEntity extends IBaseEntity {
     type: IViewType;
 }
 
-export interface IEntityCounters {
+export interface IGenericCounters {
     new: number;
```

Comment:
I think @montgomp was right about this, can we change to 'created' or 'discovered'?